src/ng-select/lib/ng-select.component.html

```diff
@@ -4,8 +4,13 @@
 	[class.ng-has-value]="hasValue"
 	class="ng-select-container">
 	<div class="ng-value-container">
-		<div class="ng-placeholder">{{ placeholder }}</div>
-
+        <ng-template #defaultPlaceholderTemplate>
+            <div class="ng-placeholder">{{ placeholder }}</div>
+        </ng-template>
+        <ng-template
+			[ngTemplateOutlet]="placeholderTemplate || defaultPlaceholderTemplate">
+        </ng-template>
```

Comment:
The placeholder stays even after selecting the item. See below

<img width="868" alt="Screenshot 2024-07-25 at 4 07 22 PM" src="https://github.com/user-attachments/assets/627536fa-5781-4f95-a0e3-5967d9245144">

Reply:
Should be fixed, also removing the placeholder div in the DOM anymore, so people can work with :before Element